samples/config/prod/README.md

```diff
@@ -18,29 +18,28 @@ Other requirements are:
 
-- The cluster should atleast have 2 nodes in the primary zone. If HA is desired then either add 2 more nodes to the primary zone or add addtional zone(s).  See the HA section below for more details on multi-zone deployment.
+- The cluster should atleast have 2 nodes in the primary zone. If HA is desired then either add 2 more nodes to the primary zone or add addtional zone(s).  See the HA section below for more details on multi-zone deployment.  The `s-cluster` and `m-cluster` samples are already designed for 2 zone HA.
 
 - Note: You can also use the `forgeops/bin/<provider>-up.sh` script to create a new cluster. For example `gke-up.sh` 
 
 > **Caution:** The sizing done in these samples is for guidance only and while it is a good starting point, you are requried to benchmark and size with your own data and use cases.
 
 
 ### HA and Multi-Zone deployments
-- If you need HA for a single zone cluster then comment out the *openamReplicaCount* and *replicas* keys in each of the yaml file in the `templates` directory.  Set the value to the desired number but make sure that your node is sized appropiately to support the number.  Also ensure the value of `ctsStores` and `ctsPassword` is accurate in amster.yaml.
+- If you need HA for a single zone cluster then comment out the *openamReplicaCount* and *replicas* keys in each of the corresponding yaml file.  Set the value to the desired number but make sure that your node is sized appropiately to support the number.  Also ensure the value of `ctsStores` and `ctsPassword` is accurate in amster.yaml.
 
-- If you need HA for multi-zone cluster then comment out the *topologyKey* in each of the yaml files.
+- If you need HA for multi-zone cluster then comment out the *topologyKey* in each of the yaml files.  See `s-cluser` and `m-cluster` yaml files as an example.
 
 - Regional HA is not supported yet by this sample. 
 
 
 ### Usage
 
-The `deploy.sh` script can take command line arguments or a configuration file.  If you choose to use the configuration directory then all you have to do is to execute `./deploy.sh -c <path to config directory>`
-
-Alternatively you can provide the variables in the configuration file via command-line switches . For information about the switches, run the script with the --help option.
+The `deploy.sh` script needs to point to a "config" directory.  There are several "config" examples provided under the `forgeops/samples` directory.  If you choose to use the existing samples, then all you have to do is to execute `./deploy.sh <path to config directory>`. For example `./deploy.sh ../samples/config/prod/m-cluster`
 
 The deployment will take anywhere from 3-10 minutes so be patient. For a successful deployment you will get a message saying "Deployment is now ready"
 
+Check out the CDM Guide on backstage.forgerock.com for more details.
```

Comment:
You probably want to remove this for now since the doc is not available on BackStage yet.

Or you could just say "Check out the CDM Cookbook in the ForgeRock documentation for more details."

Reply:
Will fix in next PR